rispy/parser.py

```diff
@@ -187,6 +187,7 @@ def load(
     mapping: Optional[Dict] = None,
     implementation: RisImplementation = RisImplementation.BASE,
     strict: bool = True,
+    **kw,
```

Comment:
does this need to change after #36 is merged in? you'll need to pop encoding off the kw stack right?

Reply:
Yeah I was planning on waiting until #36 is in before merging this